src/functions/edi/outbound/handler.ts

```diff
@@ -122,46 +111,29 @@ export const handler = async (
       },
     };
 
-    const deliveryResults = [];
+    const deliveryResults = await Promise.all(transactionSetConfig.destinations.map(
```

Comment:
Should we use `allSettled` here instead? with `all` a single failure will halt the waiting for other promises.

Reply:
My initial thought is to stick with `all` here, which is consistent with the previous implementation (any rejections within the `for` would have stopped the processing as well). 

Happy to continue the discussion and do a follow-up if needed though.

Reply:
My point was more about knowing the state of the destinations promises, say you have 3x configured - one raises an exception - `.all` will resolve immediately and the state of the other two promises would be unknown.

Reply:
Is this different than the previous implementation with the `for` loop? I suppose it is slightly different in that the other destinations would not have even started if the first one fails, and now there is a chance that one is mid-flight when a failure happens. 

The current flow feels a bit odd as-is, due to the data model. We have one input, and we generate a single envelope, but are then potentially calling map/translate N times for each destination, and then doing the delivery to those destinations. IMO this flow seems better suited for inbound (where you might want to send inbound data to multiple APIs). For outbound, it feels like we should generate a single EDI document with the envelope that has been generated, and then potentially deliver that generated document to multiple destinations (perhaps an SFTP bucket and an archive bucket, for example).

Happy to do a follow up like I said, but I think it should possibly involve a bit more than just the change to `allSettled`. Would love to hear your thoughts!